DOCSTYLE.md

```diff
@@ -0,0 +1,45 @@
+# Documentation style guidance
+
+We've defined some basic patterns to bring our documentation towards a more consistent, readable style.
+
+- We are committed to inclusive communication
+- We use British English spelling (e.g., "organisation" not "organization", "centre" not "center")
+- We prefer "-ise" over "-ize" endings
```

Comment:
redundant with en-uk point above

Reply:
agreed, this statement could be combined, I think it still stands for clarity.